specification/app/resource-manager/Microsoft.App/preview/2023-08-01-preview/ManagedEnvironments.json

```diff
@@ -1058,6 +1058,101 @@
         }
       }
     },
+    "AppInsightsConfiguration": {
+      "description": "Configuration of Application Insights ",
+      "type": "object",
+      "properties": {
+        "connectionString": {
+          "description": "Application Insights connection string",
+          "type": "string"
+        }
+      }
+    },
+    "OpenTelemetryConfiguration": {
+      "description": "Configuration of Open Telemetry",
+      "type": "object",
+      "properties": {
+        "destinationsConfiguration": {
+          "description": "Open telemetry destinations configuration",
+          "$ref": "#/definitions/DestinationsConfiguration"
+        },
+        "tracesConfiguration": {
+          "description": "Open telemetry trace configuration",
+          "$ref": "#/definitions/TracesConfiguration"
+        },
+        "logsConfiguration": {
+          "description": "Open telemetry logs configuration",
+          "$ref": "#/definitions/LogsConfiguration"
+        },
+        "metricsConfiguration": {
+          "description": "Open telemetry metrics configuration",
+          "$ref": "#/definitions/MetricsConfiguration"
+        }
+      }
+    },
+    "DestinationsConfiguration": {
+      "description": "Configuration of Open Telemetry destinations",
+      "type": "object",
+      "properties": {
+        "dataDogConfiguration": {
+          "description": "Open telemetry datadog destination configuration",
+          "$ref": "#/definitions/DataDogConfiguration"
+        }
+      }
+    },
+    "DataDogConfiguration": {
+      "description": "Configuration of datadog ",
+      "type": "object",
+      "properties": {
+        "site": {
+          "description": "The data dog site",
+          "type": "string"
+        },
+        "key": {
```

Comment:
It seems to be a secret, would you please add `x-ms-secret: true`?

refs: https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-secret